certora/conf/RecoveryModule.conf

```diff
@@ -0,0 +1,17 @@
+{
+    "files": [
+        "contracts/modules/social_recovery/SocialRecoveryModule.sol",
+    ],
+    "hashing_length_bound": "352",
+    "loop_iter": "3",
+    "msg": "RecoveryModule: General Ruleset",
+    "optimistic_hashing": true,
+    "optimistic_loop": true,
+    "process": "emv",
+    "prover_args": [
+        " -optimisticFallback true -s z3 -copyLoopUnroll 5 -mediumTimeout 5"
```

Comment:
Is there a purpose for these settings as of now? I think we should add specific flags only when needed. Same goes for `optimistic_hashing`, `optimistic_loop`, `hashing_length_bound`

Reply:
Agreed, removed all. Will add the necessary ones in #2